polyA/_runners.py

```diff
@@ -253,12 +266,10 @@ def run_full(
     active_cells[0] = [0]
 
     # add skip state pad at end
-    align_matrix[0, cols] = SKIP_ALIGN_SCORE
-    non_empty_columns.append(cols)
-    non_empty_columns.append(cols)
-    active_cells[cols] = [0]
-    active_cells[cols] = [0]
-    cols += 1
+    # TODO: I don't think we want these any more since the padding is already accounted for
+    non_empty_columns.append(column_count)
```

Comment:
Yep these non_empty_columns append statements can be removed (if non_empty_columns becomes what I mentioned above)